Unable to build

When I tried to build the ToT project following the instructions, I got:

```
$ ./gradlew assembleDebug
...
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'mobly-bundled-snippets'.
> Could not resolve all dependencies for configuration ':_debugApkCopy'.
   > Could not find com.google.android.mobly:snippetlib:0.0.1.
     Required by:
         :mobly-bundled-snippets:unspecified
```

--- a/build.gradle
+++ b/build.gradle
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.1'
+        classpath 'com.android.tools.build:gradle:2.2.3'
 
         // NOTE: Do not place your application dependencies here.
     }

--- a/build.gradle
+++ b/build.gradle
@@ -30,6 +30,6 @@ android {
 }
 
 dependencies {
-    compile 'com.google.android.mobly:snippetlib:0.0.1'
+    compile 'com.google.android.mobly:mobly-snippet-lib:1.0.0'
     compile 'com.android.support.test:runner:0.5'
 }

